quizyapp/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from typing import List
+from .models import Question, Category
+
+class MultipleQuestionsForm(forms.Form):
+    def __init__(self, question_id_list:List[str], *args, **kwargs):
+        super(MultipleQuestionsForm, self).__init__(*args, **kwargs)
+        for question_id in question_id_list:
+            question = Question.objects.get(id=question_id)
```

Comment:
Tak, możesz to też zrobić za pomocą jednego zapytania do bazy:

https://docs.djangoproject.com/en/3.1/ref/models/querysets/#in-bulk np .

Reply:
zrobione!